### DISCUSSION TITLE
对于学位论文类型，能否显示”大学“列

### DISCUSSION THREAD

**Discussion by nkh0472** [zotero-chinese/wiki#118] [求助 / Q&A] [answered]
如题，我想为学位论文创建一个视图组。
对于学位论文，大学在某种程度上是比作者还重要的信息（特定学校+年份=一个课题组的研究历程）。
但我似乎没有在自定义列中找到大学选项，对于大量学位论文只看标题和年份用来整理文献还挺痛苦的……

**Comment by zepinglee** [comment, marked as answer]
大学会显示在“publisher”一列。
<img width="175" alt="Screenshot 2023-12-26 at 21 37 46" src="https://github.com/zotero-chinese/wiki/assets/12290822/1baa4118-b95b-46fd-8659-f2faafc1f9cc">

**Reply by nkh0472** [reply]
太牛啦，我点了一遍大概是点漏了这个

**Comment by TJYYWXBG** [comment]
请问一下，学位论文条目的导师信息怎么显示出来啊
